python/_logger.py

```diff
@@ -0,0 +1,108 @@
+from enum import Enum
+from collections.abc import Iterable
+
+
+class LogLevel(Enum):
+    INFO: int = 0
+    WARNING: int = 1  # Warn in Triton in-process API
+    ERROR: int = 2
+    VERBOSE: int = 3
+
+
+class LogFormat(Enum):
+    DEFAULT: int = 0
+    ISO8601: int = 1
+
+
+# [WIP] figure out "singleton"
```

Comment:
Quick question is this still a WIP and this will be resolved before merging? is it possible to elaborate `figure out` if it is left for future work?

Reply:
This will be resolved before merging. This PR will include the actual implementation of the wrapper.